src/verifiers/sev-snp/crypto.h

```diff
@@ -0,0 +1,26 @@
+/* Copyright (c) 2022 Intel Corporation
+ * Copyright (c) 2020-2022 Alibaba Cloud
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _CRYPTO_H
+#define _CRYPTO_H
+
+#include <stdbool.h>
+#include <openssl/evp.h>
+#include <openssl/evp.h>
+#include "sevapi.h"
+#include "utils.h"
+
+typedef enum __attribute__((mode(QI))) SHA_TYPE {
+	SHA_TYPE_256 = 0,
+	SHA_TYPE_384 = 1,
+} SHA_TYPE;
+
+#define BITS_PER_BYTE 8
```

Comment:
I don't see this macro is referred by other codes except rsa_verify() so why exporting it?